### DISCUSSION TITLE
I would like to collaborate

### DISCUSSION THREAD

**Discussion by pierratono** [Learning-Plans-and-Resources/learning-plans-and-resources#11] [General]
Hello
I would love to contribute to this repository and keep help providing some AAD Learners.

My email: [pierratonoc@gmail.com](mailto:pierratonoc@gmail.com) or username: pierratono

I want to start by fixing URL in the table of content.

| Source | Error |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/39490325/128157188-0c249fd4-aca0-4b3a-94d9-d66c9a2372db.png)  | ![image](https://user-images.githubusercontent.com/39490325/128156829-172f8e64-a45a-4e39-9421-59784ac86a48.png) |

**Comment by codewithmatthew** [comment]
Hi there, @pierratono  👋.

I have sent you an invite. I have updated the links, forgot to add the `.md` extension when we changed the organization name and repository name.

Thank you for willing to help 👍.
